src/tr.i18n.js

```diff
@@ -393,22 +390,22 @@ module.exports = {
     'Komi': 'Komi',
     'Handicap': 'Handikap',
     'No stones': 'Yok',
-    '${stones} stones': p => `${p.stones} taş`,
+    '${stones} stones': p => '${p.stones} taş',
     'Board Size': 'Tahta Boyutu',
     'OK': 'Tamam',
     'Cancel': 'İptal'
   },
   'PreferencesDrawer': {
-    'Please restart ${appName} to apply your language setting.': p =>
-      `Dil ayarlarının etkin olması için ${p.appName} yeniden başlatılmalı.`,
+    'Please restart ${appName} to apply your language setting.':
+      'Dil ayarlarının etkin olması için ${p.appName} yeniden başlatılmalı.',
     'Enable hardware acceleration if possible':
-      'Kullanılabiliyorsa donanım ivmelendirmesini etkinleştir',
+      'Kullanılabiliyorsa donanım ivmesini etkinleştir',
     'Check for updates at startup': 'Başlangıçta güncellemeleri denetle',
-    'Enable sounds': 'Sesleri etkinleştir',
+    'Enable sounds': 'Ses efektlerini etkinleştir',
     'Jump to end after loading file': 'Dosya yüklendikten sonra sona atla',
-    'Fuzzy stone placement': 'Belirsiz taş yerleştirme',
-    'Animate fuzzy placement': 'Belirsiz yerleştirmeyi hareketlendir',
-    'Variation Replay Mode:': 'Varyasyon Yeniden Oynatma Modu:',
+    'Fuzzy stone placement': 'Analiz hamlelerini yerleştirme',
```

Comment:
After using program i have realized that "Belirsiz taş yerleştirme" is more proper translation.